strawberry/exceptions.py

```diff
@@ -176,3 +176,13 @@ def __init__(self, field_name: str, argument_name: str, argument_type: str):
         message = f'Argument "{argument_name}" on field "{field_name}" cannot be of type\
             "{argument_type}"'
         super().__init__(message)
+
+
+class InvalidSuperclassInterface(Exception):
+    def __init__(self, input_name: str, interface_names=List[str]):
```

Comment:
Actually, now that I think about it, wouldn't it be better to take a `List[type]` instead? Let the Exception handle everything related to output?

Reply:
You are right